src/main/java/seedu/mark/logic/commands/Command.java

```diff
@@ -20,4 +20,8 @@
      */
     public abstract CommandResult execute(Model model, Storage storage) throws CommandException;
 
+    @Override
+    public String toString() {
```

Comment:
Hm why do we need this? `Command` is an abstract class and won't be instantiated. Also, the compiler shouldn't throw an error as `Command` inherits `toString()` from `Object`.